content/chainguard/chainguard-images/working-with-images/security-advisories/how-chainguard-issues/index.md

```diff
@@ -0,0 +1,78 @@
+Chainguard Enterprise feed: https://packages.cgr.dev/chainguard/security.json 
+More information regarding these security feeds can be found at our [foundational concepts overview page](https://github.com/chainguard-dev/vulnerability-scanner-support/blob/main/docs/foundational_concepts.md) of our [vulnerability scanner support](https://github.com/chainguard-dev/vulnerability-scanner-support/tree/main) GitHub repository.
+
+If you’re wondering how these security advisories are made, you’re in the right place! In this article, we will walk through the life of a security advisory, starting from a CVE’s disclosure, all the way to its remediation. We’ll also explore what happens after an advisory is released and how its record may be updated over time.
+
+## Stages of a Security Advisory
+
+
+### Step 1: A CVE is Disclosed
+
+All security advisories begin with the disclosure of a security vulnerability. The [CVE Project](https://www.cve.org/) coordinates the processing of reported vulnerabilities through a network of CVE Numbering Authorities (CNAs). CNAs assign CVE IDs to new entries, and they are then added to a CVE Catalog. Each catalog entry contains information such as what packages or components are affected by the vulnerability, their versions, and remediation procedures, if applicable. 
+
+
+### Step 2: Scanners Detect the CVE
+
+The [National Vulnerability Database (NVD)](https://nvd.nist.gov/), the U.S. government vulnerability repository, will pick up these CVE records and review them further. During this secondary review process, the CVE entry is enriched with details that scanners later use to identify affected software. This process can take some time, so there will be issued CVEs that have not yet been analyzed by the NVD. These CVEs pending review will be marked as such by the NVD, as shown in the following image.
+
+![NVD notice stating a vulnerability has not yet been analyzed](nvd-analyze.png)
+
+In addition to the NVD, other vulnerability databases, such as the [GitHub Advisory Database](https://github.com/advisories) (GHSA) and the [Go Vulnerability Database](https://vuln.go.dev/) are also referenced by vulnerability scanners.
+
+
+### Step 3: Advisory is Issued
+
+Once a CVE has been reviewed by the NVD, it will be picked up by vulnerability scanners and reported in any affected container images. Chainguard uses [Grype](https://github.com/anchore/grype), an open-source vulnerability scanner from Anchore, as its primary tool for vulnerability detection.
+
+The newly detected CVE is then moved into the next phase where it waits for a team member to assess it. A security advisory will be issued with the status of “Under Investigation” to alert downstream users that Chainguard is aware of its presence. Security advisories are issued per package, as one CVE may impact different packages in different ways. This security advisory will be updated over time, as shown in the next step.
```

Comment:
```suggestion
The newly detected CVE is then moved into the next phase where it waits for a team member to assess it. A security advisory will be issued with the status of "Under Investigation" to alert downstream users that Chainguard is aware of its presence. Security advisories are issued per package, as one CVE may impact different packages in different ways. From there, this security advisory will be updated over time.
```

These aren't "Steps" like in a procedural tutorial, so it feels a little funky to reference the next step like this. I think this change makes it more of a regular transition sentence.

Also, I think you've got smart (aka "curly") quotes turned on!